topology/California Institute of Technology/Caltech LIGO/CIT_LIGO.yaml

```diff
@@ -149,4 +149,22 @@ Resources:
         Description: OSDF Stashcache
     AllowedVOs:
       - LIGO
+  LIGO_HANFORD_CE1:
+    Active: true
+    ContactLists:
+      Administrative Contact:
+        Primary:
+          Name: Ashton Graves
+          ID: OSG1000362
+      Security Contact:
+        Primary:
+          Name: Ashton Graves
+          ID: OSG1000362
```

Comment:
Can you add some backup contacts?

Reply:
db4e6b3444457a992002bfe70c495343efab1d27 resolves